policies.go

```diff
@@ -416,6 +417,7 @@ type tokenAwareHostPolicy struct {
 	getKeyspaceName     func() string
 
 	shuffleReplicas          bool
+	shuffleReplicasFn        func(hosts []*HostInfo) []*HostInfo
```

Comment:
Is that for testing?

Reply:
It looks like bool `shuffleReplicas` is not needed if we have this.

Reply:
It's for testing yes and perhaps we can remove the other one.

Reply:
Gone.